### DISCUSSION TITLE
Sent email automatically to business email if set on channel

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#771] [General]
Give option to user deleting comments to set it to automatically send them email with the logs of the spam comments deleted, etc.

I think this will really help if possible to send emails, cause the details of the scammer can be forwarded to other creators and those channels can be banned by those creators too.

**Comment by Firecul** [comment]
I don't have any comments I wish to get rid of on my channel, does it not save those details to a file already?

**Comment by joaogelado** [comment]
but business emails are for business, not for spam things.

**Comment by Masamune3210** [comment]
This is a horrible idea IMO, that's not what the email is meant for and would probably just be treated as spam in and of itself
